kinto_amo/utils.py

```diff
@@ -0,0 +1,14 @@
+ERROR_MSG = ("Resources should be defined as "
+             "'/buckets/<bid>/collections/<cid>'. Got %r")
+
+
+def parse_resource(resource):
```

Comment:
Maybe this should become a kinto utility function. We are using about the same in kinto-signer already.